packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -167,18 +170,16 @@ export class GuxTimeZonePickerBeta {
     );
 
     defaultZoneOptions.forEach(option => {
-      if (
-        defaultZoneOptions.length === 1 &&
-        this.workspaceDefault === this.localDefault
-      ) {
-        defaultZoneOptions[0].defaultZone = `${this.i18n(
-          'localAndWorkspaceDefault'
-        )}`;
+      if (this.workspaceDefault === this.localDefault) {
```

Comment:
i wonder if we can have the word `zone` in the `workspaceDefault` and `localDefault` props to signify that they are related to zones?

Reply:
I think this would be a breaking change if I were to change the property names.

Reply:
Ah nevermind